decidim-core/app/views/decidim/manifests/show.json.erb

```diff
@@ -2,6 +2,10 @@
   "name": "<%= organization_params.name %>",
   "lang": "<%= organization_params.default_locale %>",
   "description": "<%= organization_params.translated_description %>",
+  "display": "<%= organization_params.pwa_display %>",
+  "start_url": "/",
```

Comment:
@quinHD should this be `organization_params.start_url`??

Reply:
yep. Done!